Java/jni/native_glue.cc

```diff
@@ -17,9 +17,134 @@
 //
 
 #include "native_glue.hh"
+#include <queue>
+#include <new>
 
 using namespace litecore;
 using namespace litecore::jni;
+using namespace std;
+
+namespace litecore { namespace jni {
+        void UTF8ToModifiedUTF8(const char *input, char *output);
+        void ModifiedUTF8ToUTF8(char* input);
+        void ModifiedUTF8CharToUTF8(char* input);
+    }
+}
+
+void litecore::jni::UTF8ToModifiedUTF8(const char *input, char *output) {
```

Comment:
This confused me because it has the same name as the public function, but operates on only one character (which isn't obvious from the signature.) How about renaming it `UTF8CharToModifiedUTF8`?